### PR TITLE
Update datetime-fortran package.py

### PR DESCRIPTION
**Pull request by harshula**

### CODE REVIEW

File: packages/datetime-fortran/package.py
Comment (aidanheerdegen) [inline packages/datetime-fortran/package.py:43]: Just checking, do you want this patch to apply on when version is only `==1.7.0` or `>=1.7.0`?

Comment (harshula) [reply packages/datetime-fortran/package.py:43]: Good question. It was only tested with 1.7.0. I'll leave it to the maintainer to test other versions. ;-)

**Review by aidanheerdegen — APPROVED** [review]
Good to go. I think I'll merge.

**Comment by harshula** [comment]
For small merges like this, let's do merge with rebase. Merge with commit is just unnecessary noise.

**Pull request merged**
